includes/pages/user_shifts.php

```diff
@@ -117,23 +119,49 @@ function user_shifts() {
         $msg .= error(_('Please select a shifttype.'), true);
       }
 
-      if (isset($_REQUEST['start']) && $tmp = DateTime::createFromFormat("Y-m-d H:i", trim($_REQUEST['start'])))
+      if (isset($_REQUEST['start']) && $tmp = DateTime::createFromFormat("Y-m-d", trim($_REQUEST['start'])))
         $start = $tmp->getTimestamp();
       else {
         $ok = false;
-        $msg .= error(_("Please enter a valid starting time for the shifts."), true);
+        error(_('Please select a start date.'));
       }
-
-      if (isset($_REQUEST['end']) && $tmp = DateTime::createFromFormat("Y-m-d H:i", trim($_REQUEST['end'])))
+    
```

Comment:
Yes, I have followed the format . After every 
if{
//code
}else{
//code
}
they left a line for identation.
![screenshot from 2016-06-15 16 49 35](https://cloud.githubusercontent.com/assets/14835922/16077906/3e185930-3319-11e6-89a8-d107a663a6c3.png)